releasetool.yaml

```diff
@@ -0,0 +1,88 @@
+---
+projects:
+  vars:
+    oses:
+      - rhel7
+    signer: packages@theforeman.org
+    tar_projects: []
+    rpm_packages: []
+    koji_tags: []
+    artifact_strategy: 'rake'
+    pass_name_gpg: "theforeman/releases/foreman/{{ foreman_version }}-gpg"
+    pass_name_key: "theforeman/releases/foreman/{{ foreman_version }}-key"
+
+
+  children:
+    foreman: {}
+    katello: {}
+    client: {}
+    pulp: {}
+    keys: {}
+
+keys:
```

Comment:
I was looking for an example in the existing code that uses this information but was not finding any.

Reply:
so this was my attempt to centrally define the key, instead of setting it for each project (foreman, client, katello) individually.

it's used (indirectly) in `releasetool/data/roles/download_rpms/tasks/download.yaml`

however, I am not convinced this really solves the issue, so I am going to try a different approach.